src/Pages/Withdrawals.razor

```diff
@@ -652,12 +653,32 @@
                             if (_selectedRequest != null)
                             {
 
-                                var jobId = BackgroundJobClient.Enqueue<IBitcoinService>(service
-                                    => service.PerformWithdrawal(_selectedRequest));
+                                IScheduler scheduler = await SchedulerFactory.GetScheduler(); 
+            
+                                var map = new JobDataMap();
+                                map.Put("withdrawalRequest", _selectedRequest);
+                                
+                                var job = JobBuilder.Create<PerformWithdrawalJob>()
+                                        .DisallowConcurrentExecution()
+                                        .SetJobData(map)
+                                        .WithIdentity(nameof(PerformWithdrawalJob))
+                                        .Build();
+                                
+                                var trigger = TriggerBuilder.Create()
+                                    .WithIdentity($"{nameof(PerformWithdrawalJob)}Trigger")
+                                    .StartNow()
+                                    .WithSimpleSchedule(opts =>
+                                        opts
+                                            .WithIntervalInSeconds(10)
+                                            .WithRepeatCount(20))
+                                    .Build();
```

Comment:
I see this pattern repeated a lot, maybe it could be refactored in just a method, but its ok for now

Reply:
I added a new issue for this @RodriFS

Reply:
Excellent